src/getMessages.js

```
@@ -1,24 +1,33 @@
 'use strict';
 
 var languages = {
-	'ar-sa': require('./lang/ar-sa.json'),
-	'en-ca': require('./lang/en-ca.json'),
-	'en-gb': require('./lang/en-gb.json'),
-	'en-us': require('./lang/en-us.json'),
-	'es-mx': require('./lang/es-mx.json'),
-	'fr-ca': require('./lang/fr-ca.json'),
+	'ar': require('./lang/ar.json'),
+	'en': require('./lang/en.json'),
+	'es': require('./lang/es.json'),
+	'fr': require('./lang/fr.json'),
 	'ja': require('./lang/ja.json'),
-	'ko-kr': require('./lang/ko-kr.json'),
-	'pt-br': require('./lang/pt-br.json'),
-	'sv-se': require('./lang/sv-se.json'),
-	'tr-tr': require('./lang/tr-tr.json'),
-	'zh-cn': require('./lang/zh-cn.json'),
-	'zh-tw': require('./lang/zh-tw.json')
+	'ko': require('./lang/ko.json'),
+	'pt': require('./lang/pt.json'),
+	'sv': require('./lang/sv.json'),
+	'tr': require('./lang/tr.json'),
+	'zh-cn': require('./lang/zh-CH.json'),
+	'zh': require('./lang/zh.json'),
+	'zh-tw': require('./lang/zh-TW.json')
 };
 
 function getMessages(locale) {
+	var defaultLang = languages['en'];
 	var localeLower = typeof locale === 'string' ? locale.toLowerCase() : locale;
```

What's the desired behaviour for when something other than a string is passed in? I would think it's either to return the default locale or throw a descriptive error. Right now I think it will mostly throw an error when trying to access `localeLower.length` as most things won't have a length property or if it does, it's quite unlikely to have a substring function. I would prefer to throw a descriptive error.

And a side question, where does the locale come from? Is it possible that we get passed in a list of acceptable languages?